src/TTextEdit.cpp

```diff
@@ -993,13 +994,49 @@ void TTextEdit::mouseMoveEvent( QMouseEvent * event )
         }
     }
     x = ( event->x() / mFontWidth ) - timeOffset;
-    if( ( x < 0 ) || ( y < 0 ) || ( y > (int) mpBuffer->size()-1 ) )
+    if( ( y < 0 ) || ( y > (int) mpBuffer->size()-1 ) )
```

Comment:
Went with the `if( y < 0 ) y = 0;` approach, but I'm having a weird crash now. If I drag a selection from inside the window to the area above and left of the window, it crashes Mudlet. 😒